src/settings.ts

```diff
@@ -608,7 +609,17 @@ export const SEARCH_DEBOUNCE_TIME = 200; // ms
 export const DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT = 48;
 export const DEFAULT_DISPLAYED_VIDEO_PREVIEW_COUNT = 12;
 
-export const USER_STATUSES = [UserStatus.available, UserStatus.busy];
+export const ONLINE_USER_STATUS = {
+  status: DefaultUserStatus.online,
+  color: "#53E52A",
+};
+
+export const BUSY_USER_STATUS = {
+  status: DefaultUserStatus.busy,
+  color: "#F44336",
```

Comment:
same for colors here
can we import them from the `theme` obj?

Reply:
See https://github.com/sparkletown/sparkle/pull/1532#discussion_r662260379